containers/unit_tests/TestDualView.hpp

```diff
@@ -622,6 +622,16 @@ TEST(TEST_CATEGORY,
             const_dv.view<Kokkos::DefaultExecutionSpace>());
 }
 
+TEST(TEST_CATEGORY, dualview_sequential_host_init) {
```

Comment:
It would probably make sense to consider in the test not only a case with a type like `int` but also a case with a type with a non-trivial default constructor. E.g. a "dual view of views". To make sure that we trigger checks that the destructor is not called in a parallel region.